<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>899</width>
    <height>595</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>motion-simulation</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Tab 1</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="RenderArea" name="renderArea" native="true">
           <property name="minimumSize">
            <size>
             <width>605</width>
             <height>410</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>605</width>
             <height>410</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QTextEdit" name="textEdit">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QSlider" name="horizontalSlider">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="sliderPosLabel">
           <property name="text">
            <string>TextLabel</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="1">
          <widget class="QPushButton" name="startButton">
           <property name="text">
            <string>Start</string>
           </property>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="QComboBox" name="simCombo"/>
         </item>
         <item row="0" column="2">
          <widget class="QPushButton" name="pauseButton">
           <property name="text">
            <string>Pause</string>
           </property>
          </widget>
         </item>
         <item row="0" column="3">
          <widget class="QPushButton" name="resetButton">
           <property name="text">
            <string>Reset</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QPushButton" name="simButton">
           <property name="text">
            <string>Simulate</string>
           </property>
          </widget>
         </item>
         <item row="1" column="3">
          <widget class="QPushButton" name="batchButton">
           <property name="text">
            <string>Run Batch Simulation</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QCheckBox" name="trajCheckbox">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Draw trajectory</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QPushButton" name="trajButton">
           <property name="text">
            <string>Calculate Trajectory</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Tab 2</string>
      </attribute>
      <widget class="QWidget" name="">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>9</y>
         <width>873</width>
         <height>449</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <widget class="QLabel" name="ballPosLabel">
              <property name="text">
               <string>Ball:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="traj2Button">
              <property name="text">
               <string>Compute Trajectory</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="FIRARenderArea" name="firaRenderArea" native="true">
            <property name="minimumSize">
             <size>
              <width>605</width>
              <height>410</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>605</width>
              <height>410</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QPushButton" name="startSending">
              <property name="text">
               <string>Start Sending</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="stopSending">
              <property name="text">
               <string>Stop Sending</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QPushButton" name="receiveButton">
            <property name="text">
             <string>Receive Data</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPlainTextEdit" name="receiveDataTextEdit"/>
          </item>
          <item>
           <widget class="QLineEdit" name="logFileLineEdit">
            <property name="text">
             <string>/home/robocup/motion-simulation/logs/1</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="writeLogButton">
            <property name="text">
             <string>Write Log</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="clearButton">
            <property name="text">
             <string>Clear</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>RenderArea</class>
   <extends>QWidget</extends>
   <header location="global">renderarea.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>FIRARenderArea</class>
   <extends>QWidget</extends>
   <header location="global">firarenderarea.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>horizontalSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>sliderPosLabel</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>735</x>
     <y>440</y>
    </hint>
    <hint type="destinationlabel">
     <x>773</x>
     <y>437</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
